xc7/utils/prjxray_form_channels.py

```diff
@@ -137,21 +275,54 @@ def add_wire_to_site_relation(
   name = ?
   AND site_type_pkey = ?""", (site_pin.name, site_types[site.type])
             )
-
             result = write_cur.fetchone()
             site_pin_pkey = result[0]
+
+            intrinsic_delay = 0
+            drive_resistance = 0
+            capacitance = 0
+
+            if site_pin.timing is not None:
+                # Use the largest intristic delay for now.
+                # This conservative on slack timing, but not on hold timing.
+                #
+                # nanosecond -> seconds
+                intrinsic_delay = site_pin.timing.delays[PvtCorner.SLOW
+                                                         ].max / 1e9
+
+                if isinstance(site_pin.timing, prjxray.tile.OutpinTiming):
+                    # milliOhms -> Ohms
+                    drive_resistance = site_pin.timing.drive_resistance / 1e3
+                elif isinstance(site_pin.timing, prjxray.tile.InpinTiming):
+                    # microFarads -> Farads
+                    capacitance = site_pin.timing.capacitance / 1e6
+                else:
+                    assert False, site_pin
+            else:
+                # Use min value instead of 0 to prevent
+                # VPR from freaking out over a zero net delay.
+                intrinsic_delay = 2**-126
```

Comment:
I guess it would be good to set driver_resistance and capacitance also to 2**-126 (or other)

Reply:
No, that is not what is important.